src/Components/News/NewsListItem.css

```diff
@@ -0,0 +1,37 @@
+.card {
+    border: none;
+    box-shadow: 0px 3px 38px -13px rgba(0, 0, 0, 0.2);
+}
+
+.card-columns .card {
+    margin-bottom: 40px;
+}
+
+.card-title a {
+    color: #303133;
+}
+
+.card-title a:hover,
+.card-title a:active,
+.card-title a:focus {
+    color: #4aca85;
+}
+
+.card-subtitle {
+    color: #303133;
+    font-size: 12px;
```

Comment:
Don't use `px` in font-size and margin unless is strictly necessary, `px` doesn't scale as goods as `em` or `rem`. Since our body is `16px`, if you want `12px` do: `12/16 = 0.75` and use `rem`